### DISCUSSION TITLE
What is the latest stable quartz-scheduler version?

### DISCUSSION THREAD

**Discussion by kumaranae** [quartz-scheduler/quartz#1285] [General]
In Mvn repository, we could see the stable version released is 2.4.0 and its released on Nov 2024.
Where https://www.quartz-scheduler.org/downloads/ here it shows 2.3.0 as released stable version.

Can anyone here pls confirm which is the stable released version?

**Comment by AleMac17** [comment]
Same question, which is the stable released version right now?

**Reply by mprins** [reply]
2.5.1, see https://www.quartz-scheduler.org/downloads/
